src/ct/ct_menu.h

```diff
@@ -56,11 +56,11 @@ class CtMenu
 
 public:
     const char*       None       = "";
-    const std::string KB_CONTROL = "<control>";
-    const std::string KB_SHIFT   = "<shift>";
-    const std::string KB_ALT     = "<alt>";
-    const std::string KB_META    = "<meta>";
-    
```

Comment:
This change is going to screw up all the existing custom keyboard shortcuts of the users in config.cfg
e.g.
```
[keyboard]
insert_special_char=<control><shift><alt><meta>S
```
Cannot accept this disruption, please revert it.

Reply:
undid changes in the ct_menu.h file